packages/subproviders/test/unit/eth_lightwallet_subprovider_test.ts

```diff
@@ -0,0 +1,185 @@
+import { JSONRPCResponsePayload } from '@0xproject/types';
+import * as chai from 'chai';
+import * as lightwallet from 'eth-lightwallet';
+import Web3ProviderEngine = require('web3-provider-engine');
+import RpcSubprovider = require('web3-provider-engine/subproviders/rpc');
+
+import { EthLightwalletSubprovider } from '../../src';
+import { DoneCallback } from '../../src/types';
+import { chaiSetup } from '../chai_setup';
+import { reportCallbackErrors } from '../utils/report_callback_errors';
+
+chaiSetup.configure();
+const expect = chai.expect;
+const FAKE_ADDRESS = '0x44be42fd88e22387c43ba9b75941aa3e680dae25';
+const NUM_GENERATED_ADDRESSES = 10;
+
+describe('EthLightwalletSubprovider', () => {
+    const seedPhrase: string = 'dilemma hollow outer pony cube season start stereo surprise when edit blast';
+    const salt: string = 'kvODghzs7Ff1uqHyI0P3wI4Hso4w4iWT2e9qmrWz0y4';
+    const password: string = 'supersecretpassword99';
```

Comment:
Let's move these constants above, and also use uppercase, snake-case for them. Also, TS can infer that these are strings so we can drop the `: string` type declarations.